openapi/openapi.json

```
@@ -27255,6 +27267,33 @@
         }
       }
     },
+    "com.github.openshift.api.operator.v1.CustomLogo": {
+      "description": "CustomLogo defines a configuration based on theme types for the console UI logo.",
+      "type": "object",
+      "required": [
+        "type",
+        "themes"
+      ],
+      "properties": {
+        "themes": {
+          "description": "themes is a required field that specifies the themes for the console UI logo. This field is a list of themes, each with a Type. Each theme determines whether the logo is for the dark or light or default theme of the console UI. Each item in the list must have a unique Type. When specified, there must be at least one entry and must not have greater than 2 entries.",
+          "type": "array",
+          "items": {
+            "default": {},
+            "$ref": "#/definitions/com.github.openshift.api.operator.v1.Theme"
+          },
+          "x-kubernetes-list-map-keys": [
+            "type"
+          ],
+          "x-kubernetes-list-type": "map"
+        },
+        "type": {
+          "description": "type is a required field that specifies the type of the logo for the console UI. It determines whether the logo is for the masthead or favicon. This field is an enum with valid values \"Masthead\" and \"Favicon\". Valid values are \"Masthead\" and \"Favicon\". When set to \"Masthead\", the logo will be used in the masthead or about modal of the console UI. When set to \"Favicon\", the logo will be used as the favicon of the console UI.",
```

```suggestion
          "description": "type is a required field that specifies the type of logo for the console UI. It determines whether the logo is for the masthead or favicon. This field is an enum with valid values \"Masthead\" and \"Favicon\".  When set to \"Masthead\", the logo will be used in the masthead or about modal of the console UI. When set to \"Favicon\", the logo will be used as the favicon of the console UI.",
```

I am also wondering if enum would be better written out as enumerated type here.

---

I think keeping it as 'enum' is better since that is the common name for it.